website/docs/components/alert/partials/code/component-api.md

```diff
@@ -40,4 +40,4 @@ Title, description, actions, and generic content are passed into the alert as yi
   </C.Property>
 </Doc::ComponentApi>
 
-For more details about how to invoke these contextual components see the sections ["How to use > Description"](#how-to-use-description), ["How to use > Actions"](#how-to-use-actions) and ["How to use > Generic content"](#how-to-use-generic).
\ No newline at end of file
+For more details about how to invoke these contextual components see the sections ["How to use > Description"](#how-to-use-description), ["How to use > Actions"](#how-to-use-actions) and ["How to use > Generic content"](#how-to-use-generic).
```

Comment:
do these internal anchors really exist in the page? if not, maybe better to remove them

Reply:
Good spot – we don't have these anymore, so I removed the references.